### ISSUE TITLE
  File "hunspell/hunspell.pyx", line 26, in hunspell.hunspell.copy_to_c_string UnicodeEncodeError: 'utf-8' codec can't encode characters in position 8-13: surrogates not allowed

### ISSUE THREAD

**Issue by redstoneleo** [MSeal/cython_hunspell#7]
I got the following exception after frozen my program with cx_Freeze on Linux Mint 19.1 64bit

```
root@pc:/home/i/下载/EngkuDict/build/exe.linux-x86_64-3.6# '/home/i/下载/EngkuDict/build/exe.linux-x86_64-3.6/EngkuDict' 
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/cx_Freeze-6.0b1-py3.6-linux-x86_64.egg/cx_Freeze/initscripts/__startup__.py", line 14, in run
    module.run()
  File "/usr/local/lib/python3.6/dist-packages/cx_Freeze-6.0b1-py3.6-linux-x86_64.egg/cx_Freeze/initscripts/Console.py", line 26, in run
    exec(code, m.__dict__)
  File "EngkuDict.py", line 7, in <module>
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 656, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 626, in _load_backward_compatible
  File "/home/i/下载/EngkuDict/ConfigWindow.py", line 5, in <module>
    import Utils
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 656, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 626, in _load_backward_compatible
  File "/home/i/下载/EngkuDict/Utils.py", line 41, in <module>
    hunSpell = Hunspell() # Hunspell('en_CA') ; Hunspell('en_GB-large', hunspell_data_dir='/custom/dicts/dir')
  File "hunspell/hunspell.pyx", line 128, in hunspell.hunspell.HunspellWrap.__init__
  File "hunspell/hunspell.pyx", line 110, in hunspell.hunspell.HunspellWrap._create_hspell_inst
  File "hunspell/hunspell.pyx", line 26, in hunspell.hunspell.copy_to_c_string
UnicodeEncodeError: 'utf-8' codec can't encode characters in position 8-13: surrogates not allowed
root@pc:/home/i/下载/EngkuDict/build/exe.linux-x86_64-3.6#
```

**Comment by MSeal** [comment]
Looks like https://stackoverflow.com/questions/27366479/python-3-os-walk-file-paths-unicodeencodeerror-utf-8-codec-cant-encode-s would need to be implemented here. Any chance you could make a PR?

**Comment by dzenilee** [comment]
Similarly: 
```
  File "hunspell/hunspell.pyx", line 26, in hunspell.hunspell.copy_to_c_string
UnicodeEncodeError: 'latin-1' codec can't encode character '\u201c' in position 0: ordinal not in range(256)
```

**Comment by MSeal** [comment]
I believe this is now fixed in 1.3.2 as the package now respects locales when loading dictionaries where it was before forcing utf-8.
